simulation/tests/it/curp/mod.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty mod?

Reply:
This is a placeholder for now, the files will be added in the second PR.